modules/conversion-center/controls/hover-animation-contact-buttons.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Controls;
+
+use Elementor\Control_Hover_Animation;
+
+class Hover_Animation_Contact_Buttons extends Control_Hover_Animation {
+
+	const TYPE = 'hover_animation_contact_buttons';
```

Comment:
@nicoladj77 

- Make the name more generic, so we can use it throughout.
- Also, maybe make the default an empty array, that would give the developer a chance to specify what options they want when adding the control, wdyt?